ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbarManager.java

```diff
@@ -84,6 +85,15 @@ public void setTitle(ReactToolbar view, @Nullable String title) {
     view.setTitle(title);
   }
 
+  @ReactProp(name = "rtl")
+  public void setLayoutDirection(ReactToolbar view, Boolean flag) {
+    if (flag.equals(true)) {
+      view.setLayoutDirection(LayoutDirection.RTL);
+    } else {
+      view.setLayoutDirection(LayoutDirection.LTR);
```

Comment:
Still, shouldn't we use `LayoutDirection.LOCALE` when it's set to `null` (what is the default value?)